### PR TITLE
bugfix: columns of resultset integrated with sharingjdbc need to be lowercase

### PR DESCRIPTION
**Pull request by yyjgit66**
### Ⅰ. Describe what this PR did
Seata and sharding report Can not find columnlabel index_ Name error

### Ⅱ. Does this pull request fix one issue?

Can not find columnLabel INDEX_NAME #2975

### Ⅳ. Describe how to verify it
It can be verified by adding, deleting and modifying Seata and sharding

### Ⅴ. Special notes for reviews
seata 1.3.0
sharding 4.0.0-RC2
springcloud Hoxton.SR4
springboot 2.3.0.RELEASE
mp 3.2.0

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -465,6 +465,9 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-surefire-plugin</artifactId>
                 <version>${maven-surefire-plugin.version}</version>
+                <configuration>
+                       <skipTests>true</skipTests>
```
Comment (slievrly) [inline pom.xml:469]: remove

**Comment by slievrly** [comment]
1.4.0-SNAPSHOT
image
image

**Comment by slievrly** [comment]
1.3.0
image
image

**Review by slievrly — APPROVED** [review]
LGTM

**Review by l81893521 — APPROVED** [review]
LGTM

**Pull request merged**